src/UKHO.S100PermitService.StubService/Stubs/HoldingsServiceStub.cs

```diff
@@ -17,12 +24,66 @@ public HoldingsServiceStub(HoldingsServiceConfiguration holdingsServiceConfigura
 
         public void ConfigureStub(WireMockServer server)
         {
+            server.Given
+                (Request.Create()
+                .WithPath(new WildcardMatcher(_holdingsServiceConfiguration.Url + "/*"))
+                .UsingGet()
+                .WithHeader("Authorization", "Bearer ", MatchBehaviour.RejectOnMatch))
+            .RespondWith(Response.Create()
+                .WithStatusCode(HttpStatusCode.Unauthorized)
+                .WithHeader("X-Correlation-ID", Guid.NewGuid().ToString())
+                .WithBody(@"{ ""result"": ""token is missing""}"));
```

Comment:
Can we read this response from a JSON file as we do with the other responses, so that everything is handled in the same way? 

Change the SetResponse() method to be SetResponseFromLicenseId() - as all callers require a license id, and this method will fail without one (400 response).

Create a new method SetResponse(), and have this return any responses we want to configure now (or later) that test aspects other than response based on license id, and have an enum parameter that lets the caller specify the response, to be read from a JSON file.

Reply:
As discussed we made suggested changes mentioned in point 1 and 2 (if anything else passed then it will return 400 response). As of now we won't doing anything on point 3. Thanks !!!